### ISSUE TITLE
Quick Loot RE

### ISSUE THREAD

**Issue by Therjinok** [tiltedphoques/Mod-Compatibility#73]
# Mod information

This mod adds a looting menu akin to the one present in Fallout 4.

#### Download link

https://www.nexusmods.com/skyrimspecialedition/mods/21085

#### Dependencies

- [SKSE](https://github.com/tiltedphoques/Mod-Compatibility/issues/2)
- [SkyUI](https://github.com/tiltedphoques/Mod-Compatibility/issues/3)
- [Address Library For SKSE Plugins](https://github.com/tiltedphoques/Mod-Compatibility/issues/5)

# Compatibility

> Mod is compatible and its features function without any apparent problems.

# Additional Information

> Players can see the changes instantly while looting the same container.

>Testing was brief so there may be some problems detected with a more thorough test.

**Comment by Crowlikedream** [comment]
Yes this works really well, especially since the game does not pause when playing together. The pop up loot menu can be configured to close automatically while in combat. After combat just hover the pointer around to get the loot menu to pop up. This can be very helpful for other players who have issues with the bodies not being in the exact location they should have dropped upon death.

**Comment by UltimateCoffee** [comment]
Minor issue when playing with a controller.
While not connected to a server, everything works as expected.
When connected to a server, D-Pad buttons don't work to scroll through items in the quick loot list.

**Comment by jmanginger98** [comment]
Crashes to desktop before main menu UI loads in.
Requirement "SkyUI" works on its own but crashes when QuickLootRE is loaded.

**Comment by FinniKey** [comment]
> Crashes to desktop before main menu UI loads in. Requirement "SkyUI" works on its own but crashes when QuickLootRE is loaded.

I am having the same issue with v2.15 of the mod. v2.12 gives a missing dll error. v8.6 loads the game properly but the mod does not work.

**Comment by shermancg** [comment]
Many quickloot mods haven't been working recently.

This Quickloot mod works for me as of December 2024, and works with Skyrim Together:
https://www.nexusmods.com/skyrimspecialedition/mods/120075

**Comment by euuii** [comment]
> Crashes to desktop before main menu UI loads in. 

The same thing happened to me

**Comment by CattBasher** [comment]
Heads up for any newcomers looking at this list: This mod is outdated and broken. So no, it's not your load order or missing dependencies.... Your game will crash before even getting to the Main Menu. I've tried alternative mods as well as patches for said mods, and it just wont work with the version needed for Skyrim Together Reborn.

**Comment by MakaveliMods** [comment]
As of 5/15/2025 Quickloot RE is working, download latest version of QuickLoot RE then download Quick Loot for 1.6.1130 by shad0wshayd3 on nexus

**Comment by kurtist04** [comment]
QuickLoot EE works:

https://www.nexusmods.com/skyrimspecialedition/mods/69980?tab=description

**Comment by jasonswearingen** [comment]
i can confirm does NOT work with SkyUI also installed.  I didn't try QuickLootRE on it's own.

**Comment by DaHollii** [comment]
Better version of this mod here https://github.com/tiltedphoques/Mod-Compatibility/issues/225
